### ISSUE TITLE
pre-commit not listed as dev dependency

### ISSUE THREAD

**Issue by Ovsyanka83** [aroberge/ideas#31]
pre-commit should be listed as a dev dependency to be installed during `python setup.py develop`. Right now it must be installed separately.

**Comment by aroberge** [comment]
You are correct. I have it installed by default on my main Python version which is what is used for git. I need to find a way to clean that up and isolate my projects better.

**Comment by Ovsyanka83** [comment]
I have a suggestion but you already know what it is :D

Sorry for annoying ya with it.
